app/coffee/LoggerSerializers.coffee

```diff
@@ -1,10 +1,22 @@
+_ = require('lodash')
+
 showLength = (thing) ->
-	"length: #{thing?.length}"
+	if thing?.length then thing.length else thing
+
+showUpdateLength = (update) ->
+	if update?.op instanceof Array
+		copy = _.cloneDeep(update)
+		copy.op.forEach (element, index) ->
+			copy.op[index].i = element.i.length if element?.i?.length?
```

Comment:
You could reuse the `showLength` helper here.

Reply:
Thanks. I'd rather keep it as-is, as I don't want to set the `d` and `i` values if they're not already present, so the logic is a bit different.